src/System.IO.MemoryMappedFiles/src/System.IO.MemoryMappedFiles.csproj

```diff
@@ -27,6 +27,24 @@
     <Compile Include="System\IO\MemoryMappedFiles\MemoryMappedViewStream.cs" />
     <Compile Include="System\IO\MemoryMappedFiles\MemoryMappedFileRights.cs" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetGroup)' == 'uap' or '$(TargetGroup)' == 'uapaot' ">
```

Comment:
Yes,  S.Linq.Expressions and S.Xml.XmlSerializer are part of the minority where the uapaot configuration is really needed.